samples/snippets/requirements.txt

```
@@ -5,7 +5,7 @@ grpcio==1.46.0
 ipython===7.31.1; python_version == '3.7'
 ipython===8.0.1; python_version == '3.8'
 ipython==8.3.0; python_version >= '3.9'
-matplotlib==3.5.2
+matplotlib==3.5.1
```

```suggestion
matplotlib==3.5.2
```

Unless 3.5.1 is required for some reason, in which case just add a comment next to it with why it's needed